apps/devhub/views.py

```diff
@@ -1750,5 +1750,14 @@ def render_agreement(request, template, next_step, step=None):
 
 
 def api_key(request):
-    """TODO: this is simply a stub to be filled in at a later date"""
-    return http.HttpResponse('Ok')
+    user_id = None
+    secret = None
+
+    if request.method == 'POST':
+        # TODO: Generate an API key
```

Comment:
This means that the API key and secret will never show and there is no javascript hooked up to the form submission to create one yet.